.github/CONTRIBUTING.md

```diff
@@ -72,19 +72,20 @@ We follow a "typical" GitHub workflow for contributing changes.
       case**.
     * Commit message contents don't need a lot of details, but make sure others
       can look back later and understand your changes and why.
-4. **[Create](https://help.github.com/articles/creating-a-pull-request/) a new
+4. Ensure that you follow the [PR requirements](#pr-requirements).
```

Comment:
It might sound silly, but I prefer writing out "pull request" for documents targeted to first-time contributors. The acronyms can be confusing if you're getting started for the first time. For anything focused on core contributors, it's usually okay to abbreviate, but I have a strong preference for writing out acronyms in things like contributing guidelines.

Sorry if this feels really picky!

Reply:
Man, you have a very good point. Will change!